### PR TITLE
one-click script for Azure

### PR DESCRIPTION
**Pull request by silentrvnning**
Script to instantiate a Taraxa node running on Azure cloud

### CODE REVIEW

**Comment by rattrap** [comment]
Thanks for this @silentrvnning.

I tested it and it only seems to work on debian-based Linux distros.
Ca we make this universal and make sure it also works on MacOS?

**Comment by silentrvnning** [comment]
Yep, modified the script and it should work on MacOS now. Added also support for the general install script, which should work for more distros. Added also some error checking for various commands.

File: scripts/one-click-Azure.sh [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+  --publisher Microsoft.Azure.Extensions \
+  --protected-settings '{"fileUris": ["https://raw.githubusercontent.com/Taraxa-project/taraxa-ops/master/scripts/ubuntu-install-and-run-node.sh"],"commandToExecute": "./ubuntu-install-and-run-node.sh"}' > /dev/null
+
+if [ $? != 0 ]; then 
+  echo "Error creating VM"
+  exit 1
+fi
+
+# open ports
+az vm open-port --resource-group ${AZ_GROUP_NAME} --name ${AZ_APP_SERVICE_NAME} --port 7777,8777,10022 > /dev/null
```
Comment (rattrap) [inline scripts/one-click-Azure.sh:166]: Ports should be: `3000,7777,8777,10002`

**Review by rattrap — APPROVED** [review]

**Review by rattrap — CHANGES_REQUESTED** [review]
Looks good! Just change the ports I mentioned and I will merge it!

**Review by rattrap — APPROVED** [review]

**Pull request merged**